addons/binding/org.openhab.binding.spotify/src/main/java/org/openhab/binding/spotify/internal/SpotifySession.java

```diff
@@ -0,0 +1,1299 @@
+            }
+
+            public String getHref() {
+                return href;
+            }
+
+            public void setHref(String href) {
+                this.href = href;
+            }
+
+            public String getId() {
+                return id;
+            }
+
+            public void setId(String id) {
+                this.id = id;
+            }
+
+            public List<Image> getImages() {
+                return images;
+            }
+
+            public void setImages(List<Image> images) {
+                this.images = images;
+            }
+
+            public String getName() {
+                return name;
+            }
+
+            public void setName(String name) {
+                this.name = name;
+            }
+
+            public String getType() {
+                return type;
+            }
+
+            public void setType(String type) {
+                this.type = type;
+            }
+
+            public String getUri() {
+                return uri;
+            }
+
+            public void setUri(String uri) {
+                this.uri = uri;
+            }
+
+        }
+
+        public class Artist {
+
+            @SerializedName("external_urls")
+            @Expose
+            private ExternalUrls externalUrls;
+            @SerializedName("href")
+            @Expose
+            private String href;
+            @SerializedName("id")
+            @Expose
+            private String id;
+            @SerializedName("name")
+            @Expose
+            private String name;
+            @SerializedName("type")
+            @Expose
+            private String type;
+            @SerializedName("uri")
+            @Expose
+            private String uri;
+
+            public ExternalUrls getExternalUrls() {
+                return externalUrls;
+            }
+
+            public void setExternalUrls(ExternalUrls externalUrls) {
+                this.externalUrls = externalUrls;
+            }
+
+            public String getHref() {
+                return href;
+            }
+
+            public void setHref(String href) {
+                this.href = href;
+            }
+
+            public String getId() {
+                return id;
+            }
+
+            public void setId(String id) {
+                this.id = id;
+            }
+
+            public String getName() {
+                return name;
+            }
+
+            public void setName(String name) {
+                this.name = name;
+            }
+
+            public String getType() {
+                return type;
+            }
+
+            public void setType(String type) {
+                this.type = type;
+            }
+
+            public String getUri() {
+                return uri;
+            }
+
+            public void setUri(String uri) {
+                this.uri = uri;
+            }
+
+        }
+
+        public class Device {
```

Comment:
All classes should have a description and author header and given the size of them a separate package to contain all the Spotify objects seems sensible.